mautic/mautic
# Builder apply buttons - form validation warning

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

This is how a validation error looks like when applying from the Page Builder:
![screen shot 2017-08-18 at 15 06 28](https://user-images.githubusercontent.com/1235442/29460114-fa7ac4c8-8426-11e7-96a8-764d2d8efef5.png)

And this is on Campagin Builder:
![screen shot 2017-08-18 at 19 21 31](https://user-images.githubusercontent.com/1235442/29469983-e212e6b2-844a-11e7-94dd-8056ab5acc16.png)


#### Steps to test this PR:
1. Try to create new and edit old Emails, Pages, Campaigns.
2. Open the builder, make a change, click the Apply button directly in the builder.
3. Refresh the page to see if the change stick.
4. Test also on invalid forms like if the page name is missing to see that the validation error shows up in the builder.

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
